<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>551</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>791</width>
    <height>551</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gBox_action">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QTabWidget" name="tab_action">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_action_sample">
      <attribute name="title">
       <string>单设备</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_action_multi">
      <attribute name="title">
       <string>多设备</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>列表</string>
    </property>
    <widget class="QListView" name="list_camera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_info">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>441</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>信息</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="from_camera_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_12">
        <property name="text">
         <string>名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="box_name">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SN编号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="box_sn">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>IP地址</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="box_address">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sDKLabel_2">
        <property name="text">
         <string>SDK类型</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="box_action_type">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_7">
        <property name="text">
         <string>登录名</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="box_user_name">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Label_8">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="box_password">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Label_9">
        <property name="text">
         <string>种类</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="box_kind">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="Label_10">
        <property name="text">
         <string>品牌</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="box_brand">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="Label_11">
        <property name="text">
         <string>型号</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="box_model">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="rTMPLabel">
        <property name="text">
         <string>RTMP流1</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="btn_rtmp1">
        <property name="text">
         <string>高清</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="rTMP2Label">
        <property name="text">
         <string>RTMP流2</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="btn_rtmp2">
        <property name="text">
         <string>普清</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="menu_add"/>
    <addaction name="menu_login"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="menu_manage">
   <property name="text">
    <string>设备管理</string>
   </property>
  </action>
  <action name="menu_add">
   <property name="text">
    <string>设备添加</string>
   </property>
  </action>
  <action name="menu_login">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="menu_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
